tests/components/notify/test_init.py

```diff
@@ -97,3 +127,202 @@ async def test_warn_template(hass, caplog):
     # We should only log it once
     assert caplog.text.count("Passing templates to notify service is deprecated") == 1
     assert hass.states.get("persistent_notification.notification") is not None
+
+
+async def test_invalid_platform(hass, caplog, tmp_path):
+    """Test service setup with an invalid platform."""
+    loaded_platform = MockNotifyPlatform()
+    mock_platform(hass, "testnotify.notify", loaded_platform)
+    integration = hass.data[DATA_INTEGRATIONS]["testnotify"]
```

Comment:
Please mock the integration explicitly if needed so we don't need to access hass.data here.

https://github.com/home-assistant/core/blob/0899b67578f9407c7b4143333b3fb0386a33d2eb/tests/common.py#L1137

Reply:
Bump.

Reply:
bump?

Reply:
See https://github.com/home-assistant/core/pull/68451/files#diff-5cdb4201c1969359c13bab1f804731b15f6ab0abc59810f7252d296eab01a827R31-R44

Reply:
Not needed any more and removed.